docs/notebooks/example_dnn.ipynb

```diff
@@ -7,7 +7,8 @@
    "source": [
     "This notebooks demonstrate how to split data to train-test execute parallel DNN trainings.\n",
     "\n",
-    "The example dataset `./example1_data.zarr/` can be generated using this [Jupyter Notebook](https://vegewaterdynamics.github.io/motrainer/notebooks/example_daskml/)."
+    "The example dataset `./example1_data.zarr/` can be generated using the following Jupyter Notebook:\n",
+    "- [Covert a nested DataFrame to a Dataset](../example_read_from_one_df)"
```

Comment:
```suggestion
    "- [Covert a nested DataFrame to a Dataset](./example_read_from_one_df.ipynb)"
```

Reply:
Hi @SarahAlidoost, here I am linking to the rendered webpage rather than the Notebook itself. I find the original link works. Maybe let's keep it as it was?

Reply:
After discussion, we used the absolute links in the docs site.